### DISCUSSION TITLE
When next release of 2311?

### DISCUSSION THREAD

**Discussion by KevinAS28** [MustardChef/WSABuilds#242] [General]
I Love your repo, but there are still some minor bugs, like it takes 5 minutes to start WSA after I reboot the PC forcefully, while it only took 20 seconds if I close the WSA gently. Your last release is 2309, but the official Microsoft has 2311 already, can you tell us prediction when you can do 2311 release? Thanks

**Comment by YT-Advanced** [comment]
https://github.com/YT-Advanced/WSA-Script
